.github/workflows/test-publish.yml

```
@@ -0,0 +1,30 @@
+name: Test Publishing
```

Just to match the "Publish" workflow name:

```suggestion
name: Test Publish
```